shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -159,4 +157,44 @@
 @property(nonatomic, readonly) BOOL flt_hasFirstResponderInViewHierarchySubtree;
 @end
 
+// This recognizer delays touch events from being dispatched to the responder chain until it failed
+// recognizing a gesture.
+//
+// We only fail this recognizer when asked to do so by the Flutter framework (which does so by
+// invoking an acceptGesture method on the platform_views channel). And this is how we allow the
+// Flutter framework to delay or prevent the embedded view from getting a touch sequence.
+@interface FlutterDelayingGestureRecognizer : UIGestureRecognizer <UIGestureRecognizerDelegate>
```

Comment:
Why is this one being moved to the header?

Reply:
While it's not strictly required by this change, it's an internal-only header and the bulk of all the other interface declarations are there, so really just consistency / principle of least surprise.

Reply:
> Happy to move it back if you think that's better.

Nope, just curious since I didn't see anything in the PR using it. Consistency is a good enough reason for me.